web_tool_script_3.R

```diff
@@ -17,6 +17,8 @@ set_portfolio_parameters(file_path = fs::path(par_file_path, paste0(portfolio_na
 
 set_project_parameters(file.path(working_location, "parameter_files",paste0("ProjectParameters_", project_code, ".yml")))
 
+data_location_ext <- sub("[0-9]{4}Q[0-9]", financial_timestamp, data_location_ext)
```

Comment:
```suggestion
data_location_ext <- sub("[0-9]{4}Q[0-9]", financial_timestamp, data_location_ext)
```

Same as above

Reply:
your suggestion didn't come through here?

Reply:
I changed it equivalently